.github/workflows/cli_core_e2e_test.yaml

```diff
@@ -48,5 +48,6 @@ jobs:
         env:
           TANZU_API_TOKEN: ${{ secrets.TANZU_API_TOKEN }}
           TANZU_CLI_TMC_UNSTABLE_URL: ${{ secrets.TANZU_CLI_TMC_UNSTABLE_URL }}
+          TANZU_CLI_CEIP_OPT_IN_PROMPT_CHOICE: "Yes"
```

Comment:
Why do we need this?  It feels like it means it is missing from one if the cases in the Makefile if we need to add it here,

Reply:
Yes, I added here so that we can add this environment variable for all the e2e test, but I see your point that if developer runs the e2e test locally, it would fail. I will update the PR. Thanks